RadixWallet/Features/AssetsFeature/Components/HelperViews/ResourceBalance/ResourceBalanceView.swift

```diff
@@ -437,37 +460,68 @@ extension ResourceBalanceView {
 		}
 	}
 
+	struct Unknown: View {
+		var body: some View {
+			HStack(spacing: .small1) {
+				Image(.unknownDeposits)
+
+				Text("----")
```

Comment:
perhaps `"----"` ought to be "localized"? so that Matt can change it?

Reply:
This is a placeholder used for unknown deposits. It shouldn’t need to be changed, but if it does, it will likely require UI changes as well.